Microsoft.Toolkit.Uwp/Helpers/AppPinManager/AppPinManager.cs

```diff
@@ -0,0 +1,192 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System;
+using System.Threading.Tasks;
+using Windows.ApplicationModel.Core;
+using Windows.Foundation.Metadata;
+using Windows.System;
+using Windows.UI.Shell;
+using Windows.UI.StartScreen;
+
+namespace Microsoft.Toolkit.Uwp.Helpers
+{
+    /// <summary>
+    /// This class provides static helper methods help to add the app in startmenu or TaskBar.
+    /// </summary>
+    public static class AppPinManager
+    {
+        /// <summary>
+        /// Pin the current app in Windows TaskBar
+        /// </summary>
+        /// <returns>PinResult</returns>
+        public static async Task<PinResult> PinCurrentAppToTaskBarAsync()
+        {
+            var pinResult = PinResult.UnsupportedOs;
+
+            if (ApiInformation.IsTypePresent("Windows.UI.Shell.TaskbarManager"))
```

Comment:
if you can revert the if then return that would be great :) it is easier for maintenance and for code review.
I mean if(not) return pinResult.
this comment could be applied on all functions in this file

Reply:
If feel is not required , is this Best Practice ? , function contains multiple return statement ?

Reply:
yeah, sort of best practice because if helps eye as you don't have to track where this bracket belongs to.